rflx/specification/parser.py

```diff
@@ -533,10 +533,23 @@ def create_channel_decl(
     filename: Path,
```

Comment:
> please also have a look at other uses of aspects. For example, messages can have `Checksum` aspects and transitions in sessions can have `Desc` aspects. Maybe we have the same issue there.

See https://github.com/Componolit/RecordFlux/issues/714#issuecomment-1029089588.

Reply:
What about the Desc aspect mentioned above? Please add a unit test, even if a duplicate aspect is already detected (even as a syntax error).

Reply:
Even if it is not possible to have duplicate Desc aspects at the moment, this doesn't mean that this cannot be the case in the future. Unit test are used to prevent regressions. That is why it still makes sense to add a unit test, even if you did not change anything.